watertap/core/zero_order_costing.py

```diff
@@ -411,6 +414,146 @@ def cost_power_law_flow(blk):
             blk.unit_model.electricity[t0], "electricity"
         )
 
+    def cost_autothermal_hydrothermal_liquefaction(blk):
+        """
+        General method for costing autothermal-hydrothermal liquefaction unit. Capital cost
+        is based on the HTL reactor, booster pump and other equipments.
+        """
+        t0 = blk.flowsheet().time.first()
+
+        # Get parameter dict from database
+        parameter_dict = blk.unit_model.config.database.get_unit_operation_parameters(
+            blk.unit_model._tech_type, subtype=blk.unit_model.config.process_subtype
+        )
+
+        # Get costing parameter sub-block for this technology
+        (
```

Comment:
Would it be helpful to name these costing tech parameters based on the variable/parameter which they are associated with?

Reply:
These are intemediate variables and will not be accessed by users. If the calculations are correct, I think it is OK to use these letters. The costing for other units also directly uses letters for convenience, I know the number of these parameters in this model are more, but the calculations are similar for different equipments.